### ISSUE TITLE
cannot not find artifact org.eclipse.lyo.store:store-parent

### ISSUE THREAD

**Issue by jadelkhoury** [eclipse/lyo.store#50]
I have the following dependency to lyo-store
```
		<dependency>
			<groupId>org.eclipse.lyo.store</groupId>
			<artifactId>store-core</artifactId>
			<version>4.0.0-SNAPSHOT</version>
		</dependency>
```

I get the following error, when trying to build the project:
[ERROR] Failed to execute goal on project adaptor-polarion: Could not resolve dependencies for project com.scania.rdf:adaptor-polarion:war:0.0.1-SNAPSHOT: Failed to collect dependencies at org.eclipse.lyo.store:store-core:jar:4.0.0-SNAPSHOT: Failed to read artifact descriptor for org.eclipse.lyo.store:store-core:jar:4.0.0-SNAPSHOT: Could not find artifact org.eclipse.lyo.store:store-parent:pom:4.0.0-SNAPSHOT in lyo-releases (https://repo.eclipse.org/content/repositories/lyo-releases/) -> [Help 1]

4.0.0-SNAPSHOT does not seem to exist under https://repo.eclipse.org/content/repositories/lyo-snapshots/org/eclipse/lyo/store/store-parent/

But why is parent even needed?

**Comment by jadelkhoury** [comment]
more seriously, the problem is of course also present for 2.4.0 under https://repo.eclipse.org/content/repositories/lyo-releases/org/eclipse/lyo/store/store-parent/

**Comment by berezovskyi** [comment]
I think it's actually a self-inflicted damage. I specifically excluded the parent POM from being published. I will look into how can we fix the 2.4.0.

--
/Andrew

On 13 Dec 2018, at 09:10, Jad El-khoury <notifications@github.com<mailto:notifications@github.com>> wrote:


more seriously, the problem is of course also present for 2.4.0 under https://repo.eclipse.org/content/repositories/lyo-releases/org/eclipse/lyo/store/store-parent/

**Comment by berezovskyi** [comment]
@jadelkhoury please check; fixed in `4.0.0-SNAPSHOT`

**Comment by jadelkhoury** [comment]
@berezovskyi Store-parent still seems to be missing!

[INFO] ------------------------------------------------------------------------
[INFO] Building adaptor-oauth-webapp Maven Webapp 0.0.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] Downloading: https://repo.eclipse.org/content/repositories/lyo-snapshots/org/eclipse/lyo/store/store-parent/4.0.0-SNAPSHOT/maven-metadata.xml
[INFO] Downloaded: https://repo.eclipse.org/content/repositories/lyo-snapshots/org/eclipse/lyo/store/store-parent/4.0.0-SNAPSHOT/maven-metadata.xml (610 B at 0.5 KB/sec)
[INFO] Downloading: https://repo.eclipse.org/content/repositories/lyo-snapshots/org/eclipse/lyo/store/store-parent/4.0.0-SNAPSHOT/store-parent-4.0.0-20190121.180047-2.pom
[INFO] Downloaded: https://repo.eclipse.org/content/repositories/lyo-snapshots/org/eclipse/lyo/store/store-parent/4.0.0-SNAPSHOT/store-parent-4.0.0-20190121.180047-2.pom (4 KB at 10.1 KB/sec)
[INFO] Downloading: https://repo.eclipse.org/content/repositories/lyo-snapshots/org/eclipse/lyo/store/store-parent/4.0.0-SNAPSHOT/store-parent-4.0.0-SNAPSHOT.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 3.639 s
[INFO] Finished at: 2019-02-01T09:35:13+01:00
[INFO] Final Memory: 12M/374M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project adaptor-oauth-webapp: Could not resolve dependencies for project org.eclipse.lyo.sample:adaptor-oauth-webapp:war:0.0.1-SNAPSHOT: Could not find artifact org.eclipse.lyo.store:store-parent:jar:4.0.0-SNAPSHOT in lyo-snapshots (https://repo.eclipse.org/content/repositories/lyo-snapshots/) -> [Help 1]
[ERROR]

**Comment by jadelkhoury** [comment]
There is no store-parent*.jar to download, right? Why is it trying to do that?

**Comment by jadelkhoury** [comment]
Oops! please ignore. It was my mistake since I added a dependency to store-parent, instead of store-core.
Now it is working.

**Comment by berezovskyi** [comment]
Love errors that resolve themselves!

--
/Andrew

1 feb. 2019 kl. 09:39 skrev Jad El-khoury <notifications@github.com<mailto:notifications@github.com>>:


Closed #50<https://github.com/eclipse/lyo.store/issues/50>.
